test/uk/gov/hmrc/nisp/controllers/AccountControllerSpec.scala

```diff
@@ -199,7 +199,7 @@ class AccountControllerSpec extends UnitSpec with MockitoSugar with BeforeAndAft
 
       "return COPE page for contracted out (B) user" in {
         val result = MockAccountController.showCope()(authenticatedFakeRequest(mockUserIdContractedOut))
-        contentAsString(result) should include ("You were contracted out ")
+        contentAsString(result) should include ("Your COPE estimate is")
```

Comment:
The text that we were testing had been changed, and test was not passing, so I used some other content.